hastexo/hastexo.py

```diff
@@ -266,9 +267,12 @@ def launch_or_resume_user_stack(self, sync = False):
         kwargs = self._get_os_auth_kwargs()
         task = LaunchStackTask()
         if sync:
+            logger.info('Running synchronous launch task for [%s]' % (self.stack_name))
```

Comment:
Out of curiosity: whenever convenient, could you explain when stack launches happen synchronously, and when asynchronously?